File: EvoHtmlToPdf-v8/Samples/WinForms/VB/EvoHtmlToPdfDemo/HTML_to_PDF/PDF_Color_Spaces/Create_GrayScale_CMYK_Description.vb
Imports System.ComponentModel

Imports System.Text

Namespace EvoHtmlToPdfDemo.HTML_to_PDF.PDF_Color_Spaces
	Partial Public Class Create_GrayScale_CMYK_Description
		Inherits UserControl
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub Create_GrayScale_CMYK_Description_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			descriptionWebBrowser.Visible = True
			Dim descriptionString As String = System.IO.File.ReadAllText("DemoAppFiles\Input\Descriptions\WinForms\HTML_to_PDF\PDF_Color_Spaces\Create_GrayScale_CMYK.html")
			descriptionWebBrowser.DocumentText = descriptionString
		End Sub
	End Class
End Namespace

File: EvoHtmlToPdf-v8/Samples/WinForms/CS/EvoHtmlToPdfDemo/HTML_to_PDF/PDF_Color_Spaces/Create_GrayScale_CMYK_Description.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Data;

using System.Text;
using System.Windows.Forms;

namespace EvoHtmlToPdfDemo.HTML_to_PDF.PDF_Color_Spaces
{
    public partial class Create_GrayScale_CMYK_Description : UserControl
    {
        public Create_GrayScale_CMYK_Description()
        {
            InitializeComponent();
        }

        private void Create_GrayScale_CMYK_Description_Load(object sender, EventArgs e)
        {
            descriptionWebBrowser.Visible = true;
            string descriptionString = System.IO.File.ReadAllText(@"DemoAppFiles\Input\Descriptions\WinForms\HTML_to_PDF\PDF_Color_Spaces\Create_GrayScale_CMYK.html");
            descriptionWebBrowser.DocumentText = descriptionString;
        }
    }
}
